### DISCUSSION TITLE
Twitter

### DISCUSSION THREAD

**Discussion by noraj** [mBouamama/rawsec_cli#27] [Show and tell]
Hey matthieu, have you a twitter account? I saw none linked on [your GH profile](https://github.com/mBouamama). If you have one please give it to me because I'll prepare a communication to announce the tool.

**Comment by tyki6** [comment]
hey, @noraj: [my linkedin](https://www.linkedin.com/in/matthieubouamama/), [my twitter](https://twitter.com/tyki6TV)

**Comment by noraj** [comment]
Here we go: https://twitter.com/noraj_rawsec/status/1338209218185355264
